.github/workflows/workflow.yml

```diff
@@ -70,11 +70,11 @@ jobs:
           pip install pipenv
           pip install tox
       - name: Integration tests
-        run: tox -e py3.7 -- -m integration
+        run: tox -e py3.7 -- -rfE -m integration
```

Comment:
What does `-rfE` do?

Reply:
it shows final report on failed and error tests. make it a bit easier to know what failed

Reply:
removed, cause duplicate in tox.ini

Reply:
Still here ...

Reply:
fixed